framework/src/main/java/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -452,6 +452,10 @@ private void applyDefaults(Tree tree, AnnotatedTypeMirror type) {
 
             case IDENTIFIER:
                 elt = TreeUtils.elementFromUse((IdentifierTree) tree);
+                if (ElementUtils.isTypeDeclaration(elt)) {
```

Comment:
I'm finding that the defaulting for TypeUseLocation.TYPE_DECLARATION isn't implemented correctly. This change is a work around to fix one problem I found, but I think I'm going to have to re-write that code in a separate PR.